corehq/apps/hqadmin/tasks.py

```diff
@@ -48,3 +59,58 @@ def check_non_dimagi_superusers():
     if non_dimagis_superuser:
         _soft_assert_superusers(
             False, "{non_dimagis} have superuser privileges".format(non_dimagis=non_dimagis_superuser))
+
+
+@task(queue="email_queue", bind=True, acks_late=True)
+def send_mass_emails(self, username, recipients, subject, html, text):
+    successes = []
+    failures = []
+    for recipient in recipients:
```

Comment:
How would you feel about fetching the recipients in this task, as Manish suggested earlier?
I think pickling and passing very large arguments isn't a great practice, but it will probably work. Looking at the task in celery will be pretty ugly though.

Reply:
https://github.com/dimagi/commcare-hq/pull/20416/commits/b972a8f5692cda5a7a6ff3676ebb2424904f3931